js/ticket.js

```diff
@@ -0,0 +1,88 @@
+let place = [];
+let price = [];
+function gettingID(clickID,event){
+
+    let sliceTo = event.target.className.slice(10, 12);
+
+    if(classCheck(clickID,event)){
+        event.target.classList.toggle('selected')
+        if(event.target.className.includes('selected')){
+            if(sliceTo.endsWith(" ")){
+                sliceTo = event.target.className.slice(10, 11);
+            }
+            place.push(clickID + sliceTo);
+            price.push(highPrice(clickID));
+            blockAdder(place,clickID,event);
+        }
+        else{
+            blockAdder(place,clickID,event);
+            placeRemover(clickID,event);
+        }
+        if(price.length){
+            document.getElementById("order__total").innerHTML = price.reduce(reducer) + " AMD";
+        }
+    }
+    sectionBlock(place.length);
+    console.log(place);
+    console.log(price);
+}
+
+function reducer(total, num){
+    return total + num;
+}
+
+function seatNum(event){
+    if(event.target.className.slice(10, 12).endsWith(" ")){
+        return event.target.className.slice(10, 11);
+    }
+    return event.target.className.slice(10, 12);;
+}
+
+function blockAdder(place,clickID,event){
+    if(event.target.className.includes('selected')){
+        let list = document.getElementById("order__item");
+        let li = document.createElement("li");
+        li.innerHTML = 'Row: ' + clickID + " Seat: " + seatNum(event) + " Price: " + highPrice(clickID) + " AMD";
+        list.appendChild(li);
+    }else{
+        let list = document.getElementById("order__item");
+        for(let i = 0;i < list.childNodes.length; i++){
+            if(list.childNodes[i].innerText == 'Row: ' + clickID + " Seat: " + seatNum(event) + " Price: " + highPrice(clickID) + " AMD"){
+                list.removeChild(list.childNodes[i]);
+                break;
+            }
+        }
+    }
+}
+
+function classCheck(clickID,event){
+    if( event.target.className.includes('seat-') && !(event.target.className.includes('unable'))){
+        return true;
+    }
+}
```

Comment:
```suggestion
function classCheck(clickID,event){
    return event.target.className.includes('seat-') && !(event.target.className.includes('unable'))
}
```

Reply:
Also, move this function above where do you use it. Yes, the functions with function declaration hoist, but it can be confusing when working with the bigger code